### PR TITLE
Multiple updates per 2.0 marketing release Doc

### PR DESCRIPTION
**Pull request by jimmytipit**
General update notes:

- Homepage edits for 2.0 messaging
- FAQ updates to clarify 2.0 release
- Software page - add video, clean up columns
- Learn - clean up and messaging updates

Referencing the attached word doc:

P1 
-- Home -Integration of open source section not included until we get signoff on logo usage
-- Home - New diagram not available as yet

P3
-- Home - New blogpost not available, so not yet linked
-- Learn - I kind of disagree about adding a subnav to a page that basically links to links within the same page that only link to other pages (Videos, FAQ). Perhaps we can discuss options?

P4
-- Learn - For StarlingX Videos - I went ahead and linked straight to our YouTube playlist. Let me know what you think.
-- Learn - Echoing above, I think adding new sections that just link off to other sections is a mistake. IMO we should rely on the top navigation for this unless there is some compelling reason to highlight it within relevant text.

P5
-- Software - I added a video at the top of the page to mirror the Airship site. Let me know what you think here? Can be updated to whatever.
-- Software - Three column callouts under the first paragraphs... I'm not super happy with the layout here. Consider it a placeholder for now.  Open to suggestions.
-- Software - Have not added "latest diagram" yet. Waiting for this update.

Signed-off-by: Jimmy McArthur <jimmy@tipit.net>

### CODE REVIEW

File: site/faq/index.md [hunk 1]
```diff
@@ -64,9 +64,9 @@ StarlingX is a top-level pilot project supported by the OpenStack Foundation, bu
 
 The StarlingX team is actively working with a number of other projects and communities, both within and outside of the OpenStack Foundation projects. The community is contributing to the Nova, Horizon, Keystone and Neutron projects within OpenStack. Our teams are also involved in the OSF Edge Computing Group as well as the Akraino community and the EdgeXFoundry project.
 
-#### When is the first release of StarlingX?
+#### When is the next release of StarlingX?
 
-The first release will be in October 2018. The project will release on a 4 month cadence each March, July and November in 2019. Releases are time based.
+The first release was in October 2018. The next release is in August 2019.
```
Comment (ildikov) [inline site/faq/index.md]: There's a separate PR to update this particular answer: https://github.com/StarlingXWeb/starlingx-website/pull/41

Either here or in the other PR, I would go for an alternative and say: StarlingX 1.0 was released in October, 2018. You can now download and try out StarlingX 2.0 which was just released at the end of August, 2019. The next release is targeted towards the end of 2019 and the community is planning to keep having two releases per year onwards.

File: site/index.md [hunk 2]
```diff
@@ -63,15 +63,15 @@ getInvolvedSteps:
...
 
 <a href="/learn/" class="link is-primary">Learn More ></a>
 
 </template>
 
 <home-announcement slot="announcement" button-name="Read the blog post" link="/blog/starlingx-initial-release.html">
 
-StarlingX—the open source edge computing and IoT cloud platform optimized for low latency and high performance applications—is now available in its first release. The project was established in May as a pilot project supported by the OpenStack Foundation (OSF) and builds on code contributed by Wind River and Intel.
+StarlingX Release 2.0 is now available. The open source edge computing and IoT cloud platform optimized for low latency and high performance applications is now a container based platfom for hosting OpenStack and other cloud native applications.
```
Comment (ildikov) [inline site/index.md]: s/platfom/platform/

I wonder if we could rephrase this "a container based platform for hosting OpenStack and other cloud native applications" to show integration between OpenStack and the container platform?

File: site/.vuepress/components/hero.vue [hunk 3]
```diff
@@ -11,7 +11,11 @@
 			<div :class="['container', containerClass]">
 				<div class="hero-content">
 					<h3 class="hero-title">{{ title }}</h3><!-- /.hero-title -->
-
+					<div class="hero-subhead"
+						v-if="['Home'].includes($page.title)"
+			          >Solve the operational problem of deploying and 
+			            <br>managing distributed networks.
```
Comment (ildikov) [inline site/.vuepress/components/hero.vue]: I wonder if saying 'networks' as opposed to 'systems' or smth like makes it sound like it's primarily for Telecom use cases or it's just me who connects these two?

File: site/faq/index.md (hunk 1 above)
Comment (ildikov) [reply site/faq/index.md]: We agreed with Yong to make the changes to this question here. We would like both the question and answer to be updated as:

"What is the release cadence for StarlingX?"

"The initial (beta) StarlingX release was released in October 2018. In 2019 and moving forward, StarlingX releases will align with OpenStack releases. StarlingX will have two releases a year, following a cycle-trailing model relative to the OpenStack release cycle."

Comment (jimmytipit) [reply site/faq/index.md]: This one is complete.

File: site/index.md (hunk 2 above)
Comment (jimmytipit) [reply site/index.md]: Done.

File: site/.vuepress/components/hero.vue (hunk 3 above)
Comment (jimmytipit) [reply site/.vuepress/components/hero.vue]: Sounds like there is still some question on this? Anyone have thoughts?

Comment (ildikov) [reply site/.vuepress/components/hero.vue]: We kept networks for the two-pager document so it should be alright here too. Can double check on the call or if we forget then let's keep networks for now.

File: site/index.md (hunk 2 above)
Comment (ildikov) [reply site/index.md]: Came up with an alternative to the second sentence:

The open source edge computing and IoT cloud platform optimized for low latency and high performance applications now integrates OpenStack with container technologies to provide flexibility, robustness and support for mixed workloads.

**Pull request merged**